src/services/appointments.js

```diff
@@ -12,4 +12,26 @@ const createAppointmentService = async (body) => {
 	return appointment;
 };
 
-module.exports = { getAppointmentsByDateService, createAppointmentService };
+const getAllAppointmentsService = async ({ limit = 1, page = 1 }) => {
```

Comment:
Everything working really good except one thing... Appointments should be getting from NEWer to OLDer
![image](https://user-images.githubusercontent.com/84677177/190487145-db21a60f-713f-4164-809f-3781df94c81d.png)
but now it`s opposite, from older to newer

Reply:
Okay, so it'll be in negative order.